sdk/eventhub/event-hubs/src/eventHubSender.ts

```diff
@@ -644,14 +576,17 @@ export class EventHubSender extends LinkEntity {
           if (abortSignal) {
             abortSignal.removeEventListener("abort", onAborted);
           }
-          if (this._sender) {
```

Comment:
```
// When `removeListeners` is called on timeout, the sender might be closed and cleared	          // When `removeListeners` is called on timeout, the sender might be closed and cleared
          // So, check if it exists, before removing listeners from it.
```

The above comments are for the block of code that is now removed. Therefore, remove the comment as well

Reply:
removed